core/modules/serverroutes/get-tiddlers.js

```diff
@@ -0,0 +1,41 @@
+/*\
+title: $:/core/modules/serverroutes/get-tiddlers.js
```

Comment:
I think that this route actually gets a single tiddler, so should be singular not plural.